sdcm/cluster.py

```diff
@@ -1510,19 +1510,22 @@ def wait_for_nodes_up_and_normal(self, nodes):
 
     def get_scylla_version(self):
         if not self.nodes[0].scylla_version:
-            try:
-                result = self.nodes[0].remoter.run("rpm -q {}".format(self.nodes[0].scylla_pkg()))
-            except Exception as ex:
-                self.log.error('Failed getting scylla version: %s', ex)
-            else:
-                match = re.findall("scylla-(.*).el7.centos", result.stdout)
-                if match:
-                    scylla_version = match[0]
-                    self.log.info("Found ScyllaDB version: %s" % scylla_version)
-                    for node in self.nodes:
-                        node.scylla_version = scylla_version
+            version_commands = ["scylla --version", "rpm -q {}".format(self.nodes[0].scylla_pkg())]
+            for version_cmd in version_commands:
+                try:
+                    result = self.nodes[0].remoter.run(version_cmd)
+                except Exception as ex:
+                    self.log.error('Failed getting scylla version: %s', ex)
                 else:
-                    self.log.error("Unknown ScyllaDB version")
+                    match = re.match("(\d+[.]\d+[.]\w+).*", result.stdout)
+                    if match:
+                        scylla_version = match.group(1)
```

Comment:
Original get_scylla_version() returns version which contains the date and commit index (eg: 2018.1.2-0.20180527.c1f1556). But this PR changed it to only return the major part (eg: 2018.1.2)

Is it expected?

Reply:
I don't see any conflicts in the code to save only `major.minor.bugfix` without the rest suffix unless
I missed smth....

Reply:
Yes, it didn't break anything. We currently only use node.scylla_version in monitoring_version.